core/client/src/main/java/alluxio/client/file/options/DeleteOptions.java

```diff
@@ -24,6 +25,7 @@
 @NotThreadSafe
 public final class DeleteOptions {
   private boolean mRecursive;
+  private boolean mRemoveUFSFile;
```

Comment:
I think have two reasons leads to inconsistency, one is that ufs's file changes, and another is alluxio's file changes and do not persist to ufs. So, I will provide two options for user to choice how to fix the inconsistency problem.